sdk/resourcemanager/resources/armpolicy/zz_generated_assignments_client.go

```diff
@@ -449,7 +449,7 @@ func (client *AssignmentsClient) listHandleResponse(resp *http.Response) (Assign
 
 // NewListForManagementGroupPager - This operation retrieves the list of all policy assignments applicable to the management
 // group that match the given $filter. Valid values for $filter are: 'atScope()', 'atExactScope()' or
-// 'policyDefinitionId eq '{value}''. If $filter=atScope() is provided, the returned list includes all policy assignments
+// 'policyDefinitionId eq '{value}”. If $filter=atScope() is provided, the returned list includes all policy assignments
```

Comment:
This format is not correct. There is an [issue](https://github.com/golang/go/issues/54312) for this. But it seems no good solution for now.

Reply:
This has no impact on published documentation because godoc was already replacing `''` with `”` before 1.19. For example, see the [published documentation for this package](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armpolicy#AssignmentsClient.NewListForResourcePager). The new `gofmt` only changes comment text to match godoc's rendering. But it is annoying we can't represent the empty string as `''` in docs.

Reply:
Got it. Only `armpolicy` has such problem that we could change swagger at least.